lib/domain/app_state/search/search_interactor_state.dart

```diff
@@ -1,14 +1,49 @@
 import 'package:fluffychat/app_state/failure.dart';
+import 'package:fluffychat/app_state/initial.dart';
 import 'package:fluffychat/app_state/success.dart';
 import 'package:fluffychat/domain/model/search/search_model.dart';
+import 'package:fluffychat/presentation/model/search/presentation_search.dart';
```

Comment:
please check, nothing from `presentation` can stay in `domain`

Reply:
done